### DISCUSSION TITLE
I need a way to install blackhole-2ch v0.4.0

### DISCUSSION THREAD

**Discussion by shiva-guntoju** [ExistentialAudio/BlackHole#731] [General]
I have looked for blackhole-2ch over a lot of places including brew and github releases.

I need a way to install blackhole-2ch v0.4.0 either from brew or atleast need a .pkg file for the same. Can someone please help me with this?

**Comment by devinroth** [comment]
What version is on homebrew?

Devin

**Reply by shiva-guntoju** [reply]
0.5.0

**Comment by devinroth** [comment]
I probably have it my archives somewhere. Let me look.

Why that version specifically?

Devin

**Comment by shiva-guntoju** [comment]
Thank you @devinroth for quick response.
This version has sample rate of 44100 by default and in other versions I have to set it manually. For some reason I could not do that.

**Comment by npmstart-pray** [comment]
Why that version?  Current release is v0.5.0. 
But I believe the lead dev is trying to move everyone over to his new code, dipper. It's on his/their discord...look there for direction.

**Comment by shiva-guntoju** [comment]
I found the pkg. Thank you for the support.

**Comment by devinroth** [comment]
Awesome. Beat me to it.

Devin
